waspc/src/Wasp/ConfigFile.hs

```diff
@@ -0,0 +1,52 @@
+module Wasp.ConfigFile
+  ( discoverConfigFiles,
+    tailwindConfigFile,
+    postcssConfigFile,
+    ConfigFileRelocator (..),
+  )
+where
+
+import Data.Map (fromList)
+import qualified Data.Map as Data
+import Data.Maybe (mapMaybe)
+import StrongPath (Abs, Dir, File', Path', Rel, castRel, relfile, (</>))
+import Wasp.Common (WaspProjectDir)
+import Wasp.Generator.Common (ProjectRootDir)
+import Wasp.Generator.WebAppGenerator.Common (webAppRootDirInProjectRootDir)
+import qualified Wasp.Util.IO as Util.IO
+
+-- | A type for establishing the mapping of where to copy config files from/to.
+data ConfigFileRelocator = ConfigFileRelocator
```

Comment:
This "file relocator" is not really specific to config files. We could make this a more broadly used concept if we wanted to. But figured using it just for configs for now was probably ok too.